google/datalab/bigquery/commands/_bigquery.py

```diff
@@ -376,22 +377,54 @@ def _get_query_parameters(args, cell_body):
   if config:
     jsonschema.validate(config, BigQuerySchema.QUERY_PARAMS_SCHEMA)
 
-    # Parse query_params. We're exposing a simpler schema format than the one actually required
-    # by BigQuery to make magics easier. We need to convert between the two formats
-    parsed_params = []
-    for param in config['parameters']:
-      parsed_params.append({
-        'name': param['name'],
-        'parameterType': {
-          'type': param['type']
-        },
-        'parameterValue': {
-          'value': param['value']
-        }
-      })
-    return parsed_params
-  else:
-    return {}
+  # We merge the parameters with the airflow macros so that users can specify certain airflow
+  # macro names (like '@ds') in their sql. This is useful for enabling the user to progressively
+  # author a pipeline with %bq pipeline.
+  today = datetime.date.today()
```

Comment:
This would be slightly better, so you have just one timestamp that everything is based off of:

    today = now.date()